Berksfile

```diff
@@ -1,3 +1,5 @@
 source "https://supermarket.getchef.com"
 
 metadata
+
+cookbook 'config-driven-helper', git: 'https://github.com/inviqa/chef-config-driven-helper.git', ref: '1.4.1'
```

Comment:
It's not just testing that would require config driven helper due to the metadata file having it in.

If chef-magento-ng is used alone as a dependency of another cookbook, eg `depends 'chef-magento-ng'` with accompanying Berksfile pointer to git, this Berksfile is evaluated by the berkshelf API to grab dependencies.

This would fail to grab config-driven-helper if it isn't available on supermarket, for the same reasons that this cookbook fails without this line.

So three options that I can see:
1. Publish config-driven-helper to supermarket as you suggested.
2. Remove config-driven-helper from metadata.rb dependencies. This will require defining some default attributes so that the recipes don't bomb out on usage of `node['apache']` or `node['nginx']`.
3. Keep this line in so anything using this cookbook will fetch config-driven-helper from git too.